coriolis/conductor/rpc/server.py

```diff
@@ -316,6 +316,7 @@ def execute_replica_tasks(self, ctxt, replica_id, shutdown_instances):
         execution.id = str(uuid.uuid4())
         execution.status = constants.EXECUTION_STATUS_RUNNING
         execution.action = replica
+        execution.type = 'replica_sync'
```

Comment:
Please define constants in `constants.py` for all of these.

Reply:
Also, call it `replica_execution` to have it in line with existing terminologies.